bin/spotxBidAdapter.js

```diff
@@ -244,7 +254,8 @@ export const spec = {
                 player_height: request.video.ext.player_height,
                 content_page_url: request.video.ext.content_page_url,
                 outstream_static: request.video.ext.outstream_static,
-                ad_mute: request.video.ext.ad_mute
+                ad_mute: request.video.ext.ad_mute,
+                outstreamFunction: request.video.ext.outstreamFunction
```

Comment:
what is this?

Reply:
It's a reference to the rendering function created client side with the prebid json config.

But actually, maybe it's not needed since I am calling the renderer function there:
function outstreamRender(bid) {
  if (bid.renderer.config.outstreamFunction != null && typeof bid.renderer.config.outstreamFunction === 'function') {
    bid.renderer.config.outstreamFunction(bid);
  }
}

I am setting up the outstream function with renderer.setRender(outstreamRender); thus I think we can remove this "outstreamFunction: request.video.ext.outstreamFunction". I am going to check in my testing env. And let you know asap.

Reply:
If I'm understanding this correctly, I think my answer above applies here too. If the pub has a renderer, and we want to help them with that, that's fine, but we should assume that's not the case. I'm fine with leaving this option in here, and short circuiting the renderer that we had, but we should still support the pub not doing this, and not having a renderer.

Reply:
Same as above